src/app/Shared/Model/Api_Isolate.model.ts

```diff
@@ -23,7 +23,7 @@ export interface IsolateDTO {
             e_hly: string;
             ampc_carba_phenotype: string;
         };
-        resistences: {
+        resistance: {
```

Comment:
why is this singular now? Perhaps you should use resistances in your client model

Reply:
Done. Commit "Simplify code"

Reply:
For this change I would need to adapt every data I get from the APIs (filter, isolate, counted isoalte). I can mention it when I talk to Dominic about the API and we can change it in the BE to "resistances".

Reply:
ok, thought is was just a one-liner in a dto mapping function.